firefox-ios/Shared/KeyboardHelper.swift

```diff
@@ -46,12 +46,14 @@ public protocol KeyboardHelperDelegate: AnyObject {
     func keyboardHelper(_ keyboardHelper: KeyboardHelper, keyboardWillHideWithState state: KeyboardState)
     func keyboardHelper(_ keyboardHelper: KeyboardHelper, keyboardWillChangeWithState state: KeyboardState)
     func keyboardHelper(_ keyboardHelper: KeyboardHelper, keyboardDidChangeWithState state: KeyboardState)
+    func keyboardHelper(_ keyboardHelper: KeyboardHelper, keyboardDidHideWithState state: KeyboardState)
```

Comment:
What about using `keyboardWillHideWithState` delegate method to avoid adding new code, I don't believe would be a problem.

Reply:
I did some tests with **keyboardWillHideWithState** before to add **keyboardDidHideWithState** but there was some issues.